main.rb

```diff
@@ -0,0 +1,41 @@
+require_relative 'app'
+# menu
+
+def menu
+  puts
+  puts 'Select a option: '
+  puts '1 - List all books'
+  puts '2 - List all people'
+  puts '3 - Create a person'
+  puts '4 - Create a book'
+  puts '5 - Create a rental'
```

Comment:
- [ ] There is an issue while creating a rental, the following are the errors happened when creating:
```bash
/home/descholar/descholar/myprojects/microverse/OOP-school-library/rental.rb:12:in `initialize': undefined method `rentals' for nil:NilClass (NoMethodError)

    person.rentals << self
          ^^^^^^^^
	from /home/descholar/descholar/myprojects/microverse/OOP-school-library/app.rb:102:in `new'
	from /home/descholar/descholar/myprojects/microverse/OOP-school-library/app.rb:102:in `create_and_register_rental'
	from /home/descholar/descholar/myprojects/microverse/OOP-school-library/app.rb:111:in `create_rental'
	from main.rb:22:in `switch'
	from main.rb:36:in `block in init'
	from main.rb:33:in `loop'
	from main.rb:33:in `init'
	from main.rb:41:in `<main>'
```
Would you please look into it?

Reply:
@descholar-ceo 
Hello Descholar, please can you help me with something?, I thin that you recieve this error because you enter the ID for the person but you need enter the index, in another words you need select the index of the person list

Reply:
I put a condicional to avoid insert wrong number at create rental